remo/sdk.py

```diff
@@ -762,3 +763,61 @@ def view_annotation_stats(self, annotation_set_id: int):
 
     def _view(self, url, *args, **kwargs):
         return self.viewer.browse(self.api.url(url, *args, **kwargs))
+    
+    def generate_annotations_from_folders(self, path_to_data_folder: str, output_filename="image_classification", generate_encoding=False):
+        """
+        Creates a CSV annotation file for image classification tasks, where images are stored in folders with names matching the labels of the images. The CSV file is saved in the same input directory where images are stored. 
+        Example of data structure for a dog / cat dataset: 
+              - source_folder
+                  - dog
+                     - img1.jpg
+                     - img2.jpg
+                     - ...
+                  - cat
+                     - img199.jpg
+                     - img200.jpg
+                     - ...
+        Example:
+            data_path = "flowers_data/train"
+            remo.generate_annotations_from_folders(path_to_data_folder=data_path, output_filename="flowers", generate_encoding=True)
+            
+
+
+        Args: 
+               path_to_data_folder: path to the source folder where data is stored
+               output_filename: filename of the output annotation file
+               generate_encoding: Generate a csv mapping classes to index
+
+        Returns: 
+                CSV of Files: CSV
```

Comment:
@SreeHarshaNelaturu same comment as for Args, we should have a description of what's returned ideally